### PR TITLE
terraform: exit 0 on non-hardened environments

### PR DESCRIPTION
**Pull request by stevendanna**
Signed-off-by: Steven Danna <steve@chef.io>

### CODE REVIEW

File: terraform/test-environments/modules/chef_automate_install/main.tf [hunk 1]
```diff
@@ -113,6 +113,7 @@ EOF
   provisioner "remote-exec" {
     inline = [
       "${var.hardened_security} && sudo /home/${var.ssh_username}/soften_mounts.sh",
```
Comment (stevendanna) [inline terraform/test-environments/modules/chef_automate_install/main.tf:115]: We may want set -e here I suppose

Comment (stevendanna) [reply terraform/test-environments/modules/chef_automate_install/main.tf:115]: Let's not for now

**Review by srenatus — APPROVED** [review]
LGTM.
I wonder if we could also have an expression there? https://www.terraform.io/docs/configuration/expressions.html not a good link, but what I mean is if the var is true, render the sudo, if not, empty string or "exit 0". Less shell to deal with I suppose.

**Comment by stevendanna** [comment]
@srenatus I like that suggestion, I left it as-is since it matches a pattern used elsewhere.  Perhaps we can follow up by fixing them all at once.

**Pull request merged**